nautobot/core/templates/components/panel/body_content_text.html

```diff
@@ -1,10 +1,8 @@
 {% load helpers %}
-{%  if render_as == 'json' %}
-    {{ text_content | render_json }}
-{% elif render_as == 'yaml' %}
-    {{ text_content | render_yaml }}
-{% elif render_as == 'markdown' %}
-    {{ text_content | render_markdown }}
-{% else %}
-    {{ text_content }}
+{% if render_as == 'json' %}{{ text_content | render_json }}
+{% elif render_as == 'json_pretty' %}<pre>{{ text_content | render_json }}</pre>
+{% elif render_as == 'yaml' %}<pre>{{ text_content | render_yaml }}</pre>
+{% elif render_as == 'markdown' %}{{ text_content | render_markdown }}
+{% elif render_as == 'code' %}<pre>{{ text_content }}</pre>
+{% else %}{{ text_content }}
```

Comment:
It occurs to me that `text_content` is a bit misleading since it may be a dict or list, not necessarily a string - should it just be `value`?